applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -0,0 +1,1541 @@
+  The maximum size for ``NET_IPV4_MTU`` is 576 bytes.
+  It must not have any``NULL`` character in the middle.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XSENDTO: <size>
+
+* The ``<size>`` value is an integer and it represents the actual number of bytes sent.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XSENDTO="test.server.com",1234,"Test UDP"
+   #XSENDTO: 8
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+UDP receive data #XRECVFROM
+===========================
+
+The ``#XRECVFROM`` command allows you to receive data through the UDP channel.
+
+Set command
+-----------
+
+The set command allows you to receive data through the UDP channel.
+
+Syntax
+~~~~~~
+
+::
+
+   #XRECVFROM[=<size>]
+
+The ``<size>`` value is an integer and it represents the actual number of bytes requested.
+It is set to match the ``NET_IPV4_MTU`` when not specified.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   <data>
+   #XRECVFROM: <datatype>, <size>
+
+
+* The ``<data>`` value is a string and it contains the data being received.
+* The ``<datatype>`` parameter can accept one the following values:
+
+  * ``0`` - hexidecimal string (e.g. "DEADBEEF" for 0xDEADBEEF)
+  * ``1`` - plain text (default value)
+  * ``2`` - JSON
+  * ``3`` - HTML
+  * ``4`` - OMA TLV
+
+* The ``<size>`` value is an integer and it represents the actual number of bytes received.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#UDPRECVFROM="test.server.com",1234
+   Test OK
+   #XRECVFROM: 1, 7
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+Resolve hostname #XGETADDRINFO
+==============================
+
+The ``#XGETADDRINFO`` command allows you to resolve hostnames to IPv4 addresses.
+
+Set command
+-----------
+
+The set command allows you to resolve hostnames to IPv4 addresses.
+
+Syntax
+~~~~~~
+
+::
+
+   #XGETADDRINFO=<hostname>
+
+The ``<hostname>`` parameter is a string.
+It cannot be an IPv4 address string.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XGETADDRINFO=<ip_addr>
+
+* The ``<ip_addr>`` value is a string and it indicates the IPv4 address of the resolved hostname.
+
+Examples
+~~~~~~~~
+
+::
+
+   at#xgetaddrinfo="www.google.com"
+   #XGETADDRINFO: 172.217.174.100
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+TCP server #XTCPSVR
+===================
+
+The ``#XTCPSVR`` command allows you to start and stop the TCP server.
+
+Set command
+-----------
+
+The set command allows you to start and stop the TCP server.
+
+Syntax
+~~~~~~
+
+::
+
+   #XTCPSVR=<op>[<port>[,<sec_tag>]]
+
+
+* The ``<op>`` parameter can accept one the following values:
+
+  * ``0`` - Stop server
+  * ``1`` - Start server
+  * ``2`` - Start server with data mode support
+
```

Comment:
Could you provide some additional details on what sec_tag represents here?

Reply:
https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/cmng_set.html?cp=2_1_4_28_0
ID of the credentials that are stored on modem side, signaling modem to use specific credential

Reply:
@FrancescoSer Some more explanation about the security tag here: https://infocenter.nordicsemi.com/topic/ug_link_monitor/UG/link_monitor/lm_certificate_manager.html